ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -304,8 +304,8 @@ exampleTxBodyShelley = SL.TxBody
     (SJust metaDataHash)
   where
     -- Dummy hash to decouple from the metadata in 'exampleTx'.
-    metaDataHash :: SL.MetaDataHash StandardShelley
-    metaDataHash = SL.MetaDataHash $ mkDummyHash (Proxy @(HASH StandardCrypto)) 30
+    metaDataHash :: SL.MetadataHash StandardShelley
+    metaDataHash = SL.MetadataHash $ mkDummyHash (Proxy @(HASH StandardCrypto)) 30
```

Comment:
I would rename this local binding to `metadataHash` too.

Reply:
Fixed